src/main/java/com/google/sps/servlets/EventRetrievalServlet.java

```diff
@@ -9,7 +9,7 @@
 /**
  * This Servlet will retrieve events which will eventually be listed on home page.
  */
-@WebServlet("/")
+@WebServlet("/event-retrival")
```

Comment:
The servlet needs a name else the server's response is an error. Y'all can change the name any time.